src/components/home/features/ServiceCatalog.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { TwoColumnLayout } from 'components';
+import { createUseStyles } from 'react-jss';
+
+import BackgroundImage from '../BackgroundImage';
+
+const useStyles = createUseStyles(() => ({
+  p: {
+    fontSize: '2rem',
+  },
+}));
+
+const Text = () => {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <h2>Service catalog for discoverability</h2>
+      <p className={classes.p}>300 GitHub repos and no idea what they do?</p>
```

Comment:
I'm not sure about the line above here. I think it's the "no idea" part. I dunno I reckon customers would like to think they've some idea 😄 Nit pick so feel free to ignore; I don't have a better suggestion.

Reply:
I wonder should we mention github if it can be used for other repository servers?